docs/version/stable-release.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Stable Version
+nav_order: 1
+parent: Version
+---
+
+## Stable Release - v0.1.0
+This is the first public release of the RAPIDS Accelerator for Apache Spark. 
+The list of supported operations is provided [here](../configs.html#supported-gpu-operators-and-fine-tuning)
+
+Hardware Requirements: 
+   
+    GPU Architecture: V100 or T4 (use nvidia-smi command line utility)
```

Comment:
Agree, but we haven't tested on other GPUs such as P4, P40, etc. This is just to keep the number of issues that we address to a minimum.

Reply:
Well but what I have a P4? Should I go away and not use this plugin? If we have very narrow requirements it could mean to a third party that they need to look elsewhere.

Reply:
Users can use the plugin but I would recommend that we don't support any bugs. The reason is that we don't have an environment to reproduce the bug.

Reply:
By "unsupported" you mean that Pascal issue? I thought was a supported architecture. I may be missing context.

I am just thinking about this from the customer perspective. If I have a GPU that is not listed here I may inquire whether my GPU is supported, or if it will be soon :)  So maybe there's a slippery slope here. 

But it is a true statement that we have tested pretty extensively in these two GPUs, so it would be great to say that somehow.

Reply:
Modified the page to the following info:
Hardware Requirements: 
GPU Architecture: NVIDIA Pascal™ or better (Tested on V100 and T4 GPU)

Reply:
Thanks @krajendrannv!